### PR TITLE
Update main readme

### PR DESCRIPTION
**Pull request by AndiLi99**
Added 

- a (hopefully working) end-to-end example of how to run the project against the NYC ACE line.
- table of contents
- fleshed out About section and information on why this project was created
- documentation strategy doc

### CODE REVIEW

File: doc/docs_strategy.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Documentation Strategy
```
Comment (tim1234ltp) [inline doc/docs_strategy.md:1]: Not sure if we want this in the main branch?

Comment (AndiLi99) [reply doc/docs_strategy.md:1]: Can drop this if we feel its not needed

Comment (AndiLi99) [reply doc/docs_strategy.md:1]: I’ll remove it

**Review by kaushik12 — APPROVED** [review]
Thanks, Andi! LGTM

**Pull request merged**